windows/src/developer/TIKE/child/Keyman.Developer.UI.UfrmModelEditor.pas

```diff
@@ -161,19 +179,107 @@ function FormatToIndex(format: TLexicalModelFormat): Integer;
   Result := Ord(format) - 1;  // unknown = -1
 end;
 
+function FormatFromIndex(index: Integer): TLexicalModelFormat;
+begin
+  Result := TLexicalModelFormat(index+1);  // unknown = -1
+end;
+
 function WordBreakerToIndex(wordBreaker: TLexicalModelWordBreaker): Integer;
 begin
   Result := Ord(wordBreaker) - 1;  // unknown = -1
 end;
 
-function FormatFromIndex(format: Integer): TLexicalModelFormat;
+function WordBreakerFromIndex(index: Integer): TLexicalModelWordBreaker;
+begin
+  Result := TLexicalModelWordBreaker(index+1);  // unknown = -1
+end;
+
+type
+  TComboStringOption = record
+    value, name: string;
+    class procedure FillCombo(const opts: array of TComboStringOption;
+      combo: TComboBox); static;
+    class function GetValue(const opts: array of TComboStringOption;
+      combo: TComboBox): string; static;
+    class procedure SetValue(const opts: array of TComboStringOption;
+      const text: string; combo: TComboBox); static;
+  end;
+
+const
+  CInsertAfterWordOptions: array[0..3] of TComboStringOption = (
+    (value: ' '; name: '(space U+0020)'),
+    (value: ''; name: '(no word break)'),
+    (value: Char($0F0B); name: '(Tibetan tsheg U+0F0B)'),
+    (value: Char($1361); name: '(Ethiopian wordspace U+1361)')
```

Comment:
Would some languages want to use a zero-width space?

Reply:
Yes, probably, good idea.